neo4j/codelists.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/ONSdigital/log.go/log"
 )
 
+// Type check to ensure that Neo4j implements the driver.Observation interface
```

Comment:
comment doesnt match code

Reply:
forced pushed the fix so it didn't look like I made a copy and paste error 😄